config/_shared.json

```diff
@@ -153,6 +153,7 @@
         { "value": 459, "langSlug": "so", "name": "Afsoomaali", "wpLocale": "so_SO", "territories": [ "002" ] },
         { "value": 66, "langSlug": "sq", "name": "Shqip", "wpLocale": "sq", "territories": [ "039" ] },
         { "value": 67, "langSlug": "sr", "name": "Српски језик", "wpLocale": "sr_RS", "territories": [ "039" ] },
+        { "value": 67, "langSlug": "sr_latin", "name": "Srpski (latinica)", "wpLocale": "sr_RS", "territories": [ "039" ] },
```

Comment:
Is this a locale variant? If so we can add `"parentLangSlug": "sr",` so that Calypso knows about its relationship to the root language, as we've done for `de_formal`

Reply:
Yeah, we want to end up with `sr` for the `domLocaleSlug` [here](https://github.com/Automattic/wp-calypso/blob/588594e30b7e4cad7387b77910d50d7a5780b9de/client/lib/i18n-utils/switch-locale.js#L53).

I might use something like "htmlLang" over "parentLangSlug", though. The "parent" makes me think of the translation fallback we've got for `de_formal`, but we don't do any of that in calypso, just the `lang` (right?).

Reply:
Looks like neither `calypso.localhost` nor `calypso.live` handle the html lang :(

Reply:
Just to clarify - We update the lang tag in Javascript (obviously - I just linked that source), and "view page source" will show the original "en" in dev, but in production we're serving the page with right lang code from the get-go, so "view source" shows the right slug.

Reply:
@deBhal you can simulate the SS html lang attr population on dev by enabling user bootstrap

Reply:
> I might use something like htmlLang over parentLangSlug, though. 

👍 Whichever makes more sense, so long as we update the refs as part of this PR.

Reply:
With `"parentLangSlug": "sr",` added `html lang`:

<img width="400" alt="screen shot 2018-03-15 at 4 29 49 pm" src="https://user-images.githubusercontent.com/6458278/37445871-6f94ca6a-286e-11e8-9847-ec36ccc2320d.png">

👍